src/pages/search/registration_search/RegistrationSearchBar.tsx

```diff
@@ -117,76 +117,79 @@ export const RegistrationSearchBar = ({ registrationQuery }: Pick<SearchPageProp
           </Box>
 
           <FieldArray name="properties">
             {({ push, remove }: FieldArrayRenderProps) => (
-              <Box gridArea="advanced" sx={{ display: 'flex', flexDirection: 'column', gap: '0.75rem' }}>
-                {values.properties.map((property, index) => (
-                  <AdvancedSearchRow
-                    key={index}
-                    removeFilter={() => {
-                      remove(index);
-                      const valueToRemove = typeof property.value === 'string' ? property.value : property.value[0];
-                      const newParams = removeSearchParamValue(searchParams, property.fieldName, valueToRemove);
-                      newParams.set(ResultParam.From, '0');
-                      history.push({ search: newParams.toString() });
-                    }}
-                    baseFieldName={`properties[${index}]`}
-                  />
-                ))}
-
-                <Box sx={{ display: 'flex', gap: '1rem' }}>
-                  <Button
-                    data-testid={dataTestId.startPage.advancedSearch.addFilterButton}
-                    variant="outlined"
-                    onClick={() => {
-                      const newPropertyFilter: PropertySearch = {
-                        fieldName: '',
-                        value: '',
-                      };
-                      push(newPropertyFilter);
-                    }}
-                    startIcon={<FilterAltIcon />}>
-                    {t('search.add_filter')}
-                  </Button>
+              <>
+                <Box gridArea="filter" sx={{ display: 'flex', flexDirection: 'column', gap: '0.75rem' }}>
+                  {values.properties.map((property, index) => (
+                    <AdvancedSearchRow
+                      key={index}
+                      removeFilter={() => {
+                        remove(index);
+                        const valueToRemove = typeof property.value === 'string' ? property.value : property.value[0];
+                        const newParams = removeSearchParamValue(searchParams, property.fieldName, valueToRemove);
+                        newParams.set(ResultParam.From, '0');
+                        history.push({ search: newParams.toString() });
+                      }}
+                      baseFieldName={`properties[${index}]`}
+                    />
+                  ))}
+                </Box>
+                <Box gridArea="buttonRowBottom" sx={{ display: 'flex', alignItems: 'flex-end' }}>
                   {values.properties && values.properties.length > 0 && (
-                    <Button
-                      variant="contained"
-                      type="submit"
-                      startIcon={<SearchIcon />}
-                      data-testid={dataTestId.startPage.advancedSearch.searchButton}>
-                      {t('common.search')}
-                    </Button>
+                    <>
+                      <Button
```

Comment:
Dette ser også ut som en `IconButton`

Reply:
![image](https://github.com/BIBSYSDEV/NVA-Frontend/assets/10809754/32db1331-a73c-4c37-866f-200564f8e91d)

Reply:
hva om du endrer size til medium/large i stedet for small da?

Reply:
Button fungerer sikkert likt, bare litt wonky med `type:"text"` når den har et ikon og ikke tekst. Usikker på om det har noen UU-tilpasninger sånn spesifikt utover det 🤔

Reply:
Kjører `<IconButton />` med styling!